pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -5581,7 +5587,7 @@ def TreatedWaterWaterQualityRule(b, r, w, t):
         model.s_W,
         model.s_T,
         rule=TreatedWaterWaterQualityRule,
-        doc="Treatmed water water quality",
+        doc="Treated water water quality",
```

Comment:
```suggestion
        doc="Treated water quality",
```
While you're fixing typos....

Reply:
Good catch. Will be fixed in my next iteration.